CHANGELOG.md

```diff
@@ -7,6 +7,11 @@
   * Add `PayPalVaultResult` type to return vault result
   * Add `PayPalVaultDelegate` to handle results from vault flow
   * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
+* CardPayments
+  * Add `liabilityShift` property from `CardResult`
+  * Add `CardClientError.threeDSVerificationError` for invalid verification
+  * Add `CardClientError.missingDeeplinkgURLError` for missing deeplink URL from 3DS
```

Comment:
```suggestion
  * Add `CardClientError.missingDeeplinkURLError` for missing deeplink URL from 3DS
```

Also nitpick - since 3DS is a protocol instead of a service, `from 3DS` is a little odd to me. Maybe `from 3DS service`, or just drop the `from 3DS`.